tools/tx_pull.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+"""
+This script is meant to pull the translations from Transiflex .
+Technically, it will pull the translations from transiflex,
```

Comment:
I tries with author = {'name' : 'samuellefever', 'email' : 'sam@niboo.be'} and it stays the same